src/annotator/integrations/test/pdf-test.js

```diff
@@ -54,11 +54,15 @@ describe('PDFIntegration', () => {
     };
 
     fakePdfAnchoring = {
```

Comment:
I think it should be `fakePDFAnchoring`
```suggestion
    fakePDFAnchoring = {
```